downstream/modules/platform/ref-controller-instance-group-capacity.adoc

```diff
@@ -0,0 +1,33 @@
+[id="controller-instance-group-capacity"]
+
+= Instance group capacity limits
+
+There is external business logic that can drive the desire to limit the concurrency of jobs sent to an instance group, or the maximum number of forks to be consumed.
```

Comment:
For your consideration: note from the SSG re. "desire" - _Use 'need' rather than 'desire'._